src/Analyzers/CSharp/Analyzers/UseCollectionExpression/CollectionExpressionMatch.cs

```diff
@@ -5,7 +5,7 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.UseCollectionInitializer;
 
-namespace Microsoft.CodeAnalysis.UseCollectionExpression;
```

Comment:
a few types were in the wrong namespace.  fixed up with downstream consequences.